packages/dx-scheduler-core/src/plugins/scheduler-core/computeds.ts

```diff
@@ -17,3 +18,21 @@ export const appointments: PureComputed<
   },
   dataItem: appointment,
 }));
+
+export const dateTimeFormatComputed: DateTimeFormatComputedFn = (locale) => {
```

Comment:
Should be renamed to formatDateTimeGetter: FormatDateTimeGetterFn since it returns a function